src/commands/importExportEntities/entities/contentTypes.ts

```diff
@@ -188,3 +221,22 @@ const createUpdateTypeItemReferencesFetcher = (params: UpdateTypeParams) => (typ
     .withData(patchOps)
     .toPromise();
 };
+
+const cleanElementsFromType = (client: ManagementClient, type: Type, excludeElementTypes?: string | string[]) => {
```

Comment:
let's keep it with only array. And if you update management sdks, then you can you something like this => type ElementTypes = ContentTypeElements.Element['type']. This will allow you to specify only types that are valid :)